src/main/java/com/aws/greengrass/clientdevices/auth/metrics/ClientDeviceAuthMetrics.java

```diff
@@ -63,49 +69,35 @@ public void emitMetrics() {
     public List<Metric> collectMetrics() {
         List<Metric> metricsList = new ArrayList<>();
 
-        long timestamp = Instant.now(clock).toEpochMilli();
-
-        Metric metric = Metric.builder()
-                .namespace(NAMESPACE)
-                .name(METRIC_SUBSCRIBE_TO_CERTIFICATE_UPDATES_SUCCESS)
-                .unit(TelemetryUnit.Count)
-                .aggregation(TelemetryAggregation.Sum)
-                .value(subscribeToCertificateUpdatesSuccess.getAndSet(0L))
-                .timestamp(timestamp)
-                .build();
-        metricsList.add(metric);
+        metricsList.add(buildMetric(METRIC_SUBSCRIBE_TO_CERTIFICATE_UPDATES_SUCCESS,
+                subscribeToCertificateUpdatesSuccess));
+        metricsList.add(buildMetric(METRIC_SUBSCRIBE_TO_CERTIFICATE_UPDATES_FAILURE,
+                subscribeToCertificateUpdatesFailure));
+        metricsList.add(buildMetric(METRIC_VERIFY_CLIENT_DEVICE_IDENTITY_SUCCESS,
+                verifyClientDeviceIdentitySuccess));
+        metricsList.add(buildMetric(METRIC_VERIFY_CLIENT_DEVICE_IDENTITY_FAILURE,
+                verifyClientDeviceIdentityFailure));
+        metricsList.add(buildMetric(METRIC_AUTHORIZE_CLIENT_DEVICE_ACTIONS_SUCCESS,
+                authorizeClientDeviceActionSuccess));
+        metricsList.add(buildMetric(METRIC_AUTHORIZE_CLIENT_DEVICE_ACTIONS_FAILURE,
+                authorizeClientDeviceActionFailure));
 
-        metric = Metric.builder()
-                .namespace(NAMESPACE)
-                .name(METRIC_SUBSCRIBE_TO_CERTIFICATE_UPDATES_FAILURE)
-                .unit(TelemetryUnit.Count)
-                .aggregation(TelemetryAggregation.Sum)
-                .value(subscribeToCertificateUpdatesFailure.getAndSet(0L))
-                .timestamp(timestamp)
-                .build();
-        metricsList.add(metric);
+        return metricsList;
+    }
 
-        metric = Metric.builder()
-                .namespace(NAMESPACE)
-                .name(METRIC_VERIFY_CLIENT_DEVICE_IDENTITY_SUCCESS)
-                .unit(TelemetryUnit.Count)
-                .aggregation(TelemetryAggregation.Sum)
-                .value(verifyClientDeviceIdentitySuccess.getAndSet(0L))
-                .timestamp(timestamp)
-                .build();
-        metricsList.add(metric);
+    private Metric buildMetric(String metricName, AtomicLong metricValue) {
+        long timestamp = Instant.now(clock).toEpochMilli();
```

Comment:
there's a subtle change here, instead of all metrics having the same timestamp, they may be different now.  would advise reverting to old behavior, will be consistent w/ other emitters (KernelMetricsEmitter/SystemMetricsEmitter)